### ISSUE TITLE
Configurations of the pre-trained models

### ISSUE THREAD

**Issue by jeff024** [deeplearning-wisc/cider#2]
Hi authors,

This is a wonderful work of OOD detection. Your work is very interesting and promising. Could you please provide us the configuration you used for the provided pre-trained model checkpoints? Especially the momentum for prototypes update. Cause in the paper the momentum is set to 0.95 by default but in the training scripts the momentum is set to 0.5, which seems to provide stronger performance than trained using 0.95

Many thanks

**Comment by alvinmingwisc** [comment]
Thank you for the question. The momentum parameter is a hyperparameter that typically depends on the ID dataset. Although the default value for alpha is set as 0.95, we have observed that alpha=0.5 produces stronger results for CIFAR-100, while alpha=0.99 performs better for CIFAR-10 (as shown in Appendix C, page 16), which are the values used in the provided checkpoints.

In addition, without the momentum, the prototypes can be set as learnable (page 16, Ablation on gradient-based prototype update). We will provide a script for both EMA and learnable prototype-updates soon.
